scripts/qa-vdb

```diff
@@ -67,6 +67,10 @@ usage <<-EOU
 	Missing slots won't be displayed when a binding operator (:=) is used as
 	portage records the slot even if it was missing from the ebuild.
 
+	No warning is given in the event libraries are missing entirely or not
```

Comment:
> But... No warning is given _if one_ (or two, or all of them) library is missing. Does your text say "_if all libraries_ libraries", doesn't it?

Implied "any libraries" rater than all, but could reword it a bit to be clearer (and get rid of the event thing like at it), maybe:
```suggestion
	No warning is given for libraries that are missing entirely or not
```
Applies to any libraries that weren't found regardless of amount.

Reply:
Aha, I see: `entirely or not`. Strange but I saw `entirely` only :smile:

Reply:
Aha, I see: `entirely or not`. Strange but I saw `entirely` only :smile: 
`entirely or not, associated` vs `entirely, or not associated`, hm...

Ok, my speech was about `entirely` word.